benchmark/sparse/linear_regression/results.md

```diff
@@ -9,22 +9,24 @@
 | Dataset          | Synthetic(Randomly generated)                                |
 | :--------------- | :----------------------------------------------------------- |
 | Keras            | v2.2.2                                                      |
-| TensorFlow       | v1.9.0                                                      |
-| MXNet            | v1.2.1                                                       |
+| TensorFlow       | v1.10.0                                                      |
+| MXNet            | v1.3.0                                                      |
 
 
 ### Results 
-| Instance Type | GPUs  | Batch Size  | MXNet (Time/Epoch) | Keras-TensorFlow (Time/Epoch)  |
-|-----|-----|-----|----- |-----|
-|  C5.18X Large |   0  | 16  | 51.15 sec | 71.85 sec |
-|  C5.18X Large |   0  | 32 | 20.82 sec | 54.31 sec |
-|  C5.18X Large |   0  | 64  | 13.13 sec | 19.20 sec |
-|  C5.18X Large |   0  | 128  | 5.72 sec | 9.86 sec |
+##### Sparse data
+###### Using 25 epochs
+| Instance Type | GPUs  | Batch Size  | MXNet (Time/Epoch) | Keras-MXNet (Time/Epoch) | Keras-TensorFlow (Time/Epoch)  |
+|-----|-----|-----|-----|-----|-----|
+| C5.18XLarge |   0  | 16  | 52.89 sec | 334.32 sec | 237.28 sec |
```

Comment:
Can we report an average of a few runs, not a fixed seed result

Reply:
done